src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerControllerTest.java

```diff
@@ -771,6 +776,35 @@ public void dynamicRestrictedReportingErrors() throws Exception {
                 .andExpect(jsonPath("$.warnings", nullValue()));
     }
 
+    @Test
+    public void reinstateClosedCaseMidEventValidation() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(true);
+        when(caseCloseService.validateReinstateClosedCaseMidEvent(isA(CaseData.class))).thenReturn(anyList());
+        mvc.perform(post(REINSTATE_CLOSED_CASE_MID_EVENT_VALIDATION_URL)
+                .content(requestContent.toString())
+                .header("Authorization", AUTH_TOKEN)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.data", notNullValue()))
+                .andExpect(jsonPath("$.errors", notNullValue()))
+                .andExpect(jsonPath("$.warnings", nullValue()));
+    }
+
+    @Test
+    public void reinstateClosedCaseMidEventValidationValidationErrors() throws Exception {
```

Comment:
How about this test reinstateClosedCaseMidEventValidationErrors

Reply:
Thank you Tensay. Updated.